<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>project_tamplate</class>
 <widget class="QMainWindow" name="project_tamplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>426</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>718</width>
    <height>426</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>718</width>
    <height>426</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Новый проект на Gydrium Engine</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(46, 47, 48);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="title">
     <string>Основные параметры нового проекта</string>
    </property>
    <widget class="QLineEdit" name="PathToProjectQLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Выберите директорию для сохранения вашего проекта</string>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="inputMask">
      <string notr="true"/>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="maxLength">
      <number>32767</number>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="SaveprojectInLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Сохранить проект в:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ProjectDirectorySelectionPushButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>90</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Выбрать</string>
     </property>
    </widget>
    <widget class="QLabel" name="WorldTypeLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Выберите тип мира</string>
     </property>
     <property name="text">
      <string>Тип мира:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ChooseTypeWorldQComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Открытый</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ограниченный</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="ProjectName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Имя проекта:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ProjectNameQLineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>361</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Выберите имя проекта</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ProgrammingLanugageLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Язык программирования:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="ChooseProgrammingLanguageQComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>361</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>C++ Fork Lang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C# Fork Lang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Java Fork Lang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Loli Fork Lang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Python Fork Lang</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="UseStandartObjectsQCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>251</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Использовать стандартные объекты.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="BuildStartSceneQCheckBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Построить начальную сцену.</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>311</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Короткая настройка перед входом в редактор</string>
    </property>
    <widget class="QRadioButton" name="GraphicsMinQRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Минимальная</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="GraphicsMiddleQRadioButton">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Средняя</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="GraphicsMaxQRadioButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Максимальная</string>
     </property>
    </widget>
    <widget class="QLabel" name="RenderLevelLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Уровень рендера и пост обработки</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>200</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Справка</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>291</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <widget class="QTextBrowser" name="InfoObjectQTextBrowser">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>291</width>
         <height>161</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="ClearPushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelPushButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Отмена</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ApplyPushButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Применить</string>
    </property>
   </widget>
   <zorder>ClearPushButton</zorder>
   <zorder>ApplyPushButton</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>CancelPushButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ClearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>ProjectNameQLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>408</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ClearPushButton</sender>
   <signal>clicked()</signal>
   <receiver>PathToProjectQLineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>472</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>307</x>
     <y>100</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CancelPushButton</sender>
   <signal>clicked()</signal>
   <receiver>project_tamplate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>642</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>393</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>tuggled(bool)</signal>
  <slot>s_Tuggle(bool)</slot>
 </slots>
</ui>
